docs/_docs/continuous-integration/circleci.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "CircleCI"
+---
+
+Building, testing, and deploying your Jekyll generated website can quickly be done with [CircleCI][0]. [GitHub][1] and [Bitbucket][2] are supported and you can get started with an open-source or private repository for free.
```

Comment:
"CircleCI supports GitHub and Bitbucket, and you can get started for free using an open-source or private repository" (eliminates passive voice)